### PR TITLE
Read "MULTI-VALUED" attribute for "ATTRIBUTE-DEFINITION-ENUMERATION"

### PR DESCRIPTION
**Pull request by bidolah**

### CODE REVIEW

File: ReqIFSharp/AttributeDefinition/AttributeDefinitionEnumeration.cs [hunk 1]
```diff
@@ -115,6 +115,11 @@ public override void ReadXml(XmlReader reader)
         {
             base.ReadXml(reader);
 
+            if (reader.GetAttribute("MULTI-VALUED") == "true")
+            {
+                IsMultiValued = true;
```
Comment (alexatrhea) [inline ReqIFSharp/AttributeDefinition/AttributeDefinitionEnumeration.cs]: Please use `this.` qualifier for all public and private property, field and method usages for consistency.

Comment (bidolah) [reply ReqIFSharp/AttributeDefinition/AttributeDefinitionEnumeration.cs]: Done.

File: ReqIFSharp/AttributeDefinition/AttributeDefinitionEnumeration.cs [hunk 2]
```diff
@@ -115,6 +115,11 @@ public override void ReadXml(XmlReader reader)
         {
             base.ReadXml(reader);
 
+            if (reader.GetAttribute("MULTI-VALUED") == "true")
```
[the remark this answers is not in the document]
Comment (bidolah) [reply ReqIFSharp/AttributeDefinition/AttributeDefinitionEnumeration.cs]: image

Does this look good for you?

[the remark this answers is not in the document]
Comment (samatrhea) [reply ReqIFSharp/AttributeDefinition/AttributeDefinitionEnumeration.cs]: sure, lets go ahead and do it that way

**Pull request merged**